site/docs/reference/Connectors/materialization-connectors/PostgreSQL.md

```diff
@@ -0,0 +1,81 @@
+
+This connector materializes Flow collections into tables in a PostgreSQL database.
+
+[`ghcr.io/estuary/materialize-postgres:dev`](https://ghcr.io/estuary/materialize-postgres:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* An existing catalog spec that includes at least one collection.
+* A Postgres database to which to materialize, and user credentials.
+  Optionally, you can choose existing tables to materialize to; otherwise, the tables you specify will be created by the connector.
```

Comment:
There was a discussion on the Firebolt PR, my understanding of which is that materialize-ostgres always tries to create a table unless materialize-postgres itself has made the table before, which means it does not allow using of a table made by users, see the discussion here: https://github.com/estuary/flow/pull/407#discussion_r834579173

So I think we might have to edit this part to say that the connector creates the table unless. cc @jgraettinger

Reply:
Thank you for pointing this out - I'm sure that's correct, I was making a bit of an assumption here.